packages/nx/src/generators/utils/glob.ts

```diff
@@ -0,0 +1,63 @@
+import { Tree } from '../tree';
+import { combineGlobPatterns } from '../../utils/globs';
+import { workspaceRoot } from '../../utils/workspace-root';
+
+import minimatch = require('minimatch');
+
+/**
+ * Performs a glob search on the files in a workspace. Paths should be unix-style
+ * with forward slashes.
+ *
+ * @param patterns A list of glob patterns
+ * @returns Normalized paths in the workspace that match the provided glob patterns.
+ */
+export async function glob(patterns: string[]): Promise<string[]>;
```

Comment:
I'm uneasy about exposing this version of it. Why use this instead of `fast-glob` or some minimal wrapper of `globset`? Anyways, I'd say we shouldn't do this until someone asks for it.

Reply:
You'd use this over `fast-glob` or `globset` when the intention is to be consistent with how globs run for the rest of Nx, I think its pretty valuable and I'd likely use it for nx-dotnet.